cherrypy/tutorial/tut06_default_method.py

```diff
@@ -36,10 +40,13 @@ def index(self):
 
     @cherrypy.expose
     def default(self, user):
-        # Here we react depending on the virtualPath -- the part of the
-        # path that could not be mapped to an object method. In a real
-        # application, we would probably do some database lookups here
-        # instead of the silly if/elif/else construct.
+        """Handle Default for UsersPage.
+
+        Here we react depending on the virtualPath -- the part of the
+        path that could not be mapped to an object method. In a real
+        application, we would probably do some database lookups here
+        instead of the silly if/elif/else construct.
+        """
```

Comment:
```suggestion
        """Produce HTTP response body of the users app fallback URI."""
        # Here we react depending on the virtualPath -- the part of the
        # path that could not be mapped to an object method. In a real
        # application, we would probably do some database lookups here
        # instead of the silly if/elif/else construct.
```